src/js/components/Brick.js

```diff
@@ -0,0 +1,56 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+
+const CLASS_ROOT = "brick";
+
+export default class Brick extends Component {
+  render() {
+    let classes = [CLASS_ROOT];
+    classes.push(`${CLASS_ROOT}--${this.props.width}-${this.props.height}`);
+
+    if (this.props.colorIndex) {
+      classes.push('background-color-index-' + this.props.colorIndex);
```

Comment:
we can use string template here as well. `background-color-index-${this.props.colorIndex}`